mobile/src/Leaderboard.js

```diff
@@ -32,9 +32,12 @@ export default class Leaderboard extends PureComponent {
             <Text style={i === 0 ? s.name2 : s.name} numberOfLines={1}>
               {p.user.firstName} {p.user.lastName}
             </Text>
-            <Text style={i === 0 ? s.points2 : s.points}>
-              {p.score} {p.score === 1 ? 'pt' : 'pts'}
-            </Text>
+            <View style={s.column}>
+              <Text style={i === 0 ? s.points2 : s.points}>
+                {p.score} {p.score === 1 ? 'pt' : 'pts'}
+              </Text>
+              <Text style={s.time}>Avg {p.time ? (p.time / p.score / 1000) * -1: 0} s</Text>
```

Comment:
Same question here about multiplying by `-1` and formatting to tenths of a second.